src/users/usersReducers.js

```diff
@@ -29,9 +32,15 @@ export default (state: UsersState = initialState, action: Action): UsersState =>
     case ACCOUNT_SWITCH:
       return initialState;
 
-    case INIT_USERS:
-      return action.users.map(mapApiToStateUser);
+    case INIT_USERS: {
+      const users = action.users.map(mapApiToStateUser);
 
+      // TODO: improve and add tests
```

Comment:
We are replacing state intentionally.
The state will very likely differ.
In case of difference, the cost will be double = deep comparison + repaint.